.github/workflows/packages.yml

```diff
@@ -7,6 +7,7 @@ on:
   push:
     branches:
       - 'packages-upload-new/**'
+  workflow_dispatch:
```

Comment:
I don't think we want this -- this is quite a special-purpose workflow that uploads packages to the artifactory.